src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 0.3s;
```

add all transitioned properties

```suggestion
    transition: transform 0.3s, color 0.3s;
```